admin_manual/release_notes.rst

```diff
@@ -16,6 +17,102 @@ Release Notes
+
+New config option to enable fallback to HTTP for federated shares
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+For security reasons federated sharing (sharing between different ownCloud instances) strictly requires HTTPS (SSL/TLS). When this behavior is undesired the insecure fallback to HTTP needs to be enabled explicitly by setting ``'sharing.federation.allowHttpFallback' => false,`` to ``true`` in *config.php*.
+
+Migration related to auth_tokens (app passwords)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Upgrading to 10.0.8 includes migrations related to *auth_tokens* (*app passwords*). When users create *app passwords* as separate passwords for their clients the upgrade duration will increase depending on user count. Please consider this when planning the upgrade.
+
+Changed behavior of e-mail autocomplete for public link share dialog
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When the *'Sharing'* settings option ``Allow users to send mail notifications for shared files`` for public links is enabled, users can send public links via mail from within the web interface. The behavior of the autocomplete when entering mail addresses in the public link share dialog has been changed. Previously the autocomplete queried for local users, users from federated address books and contacts from CardDAV/Contacts App. As public links are not intended for sharing between ownCloud users (local/federated), those have been removed. Contacts synchronized via CardDAV or created in the Contacts app will still appear as suggestions.
+
+Global option for CORS domains
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+For security reasons ownCloud has a *Same-Origin-Policy* that prevents requests to ownCloud resources from other domains as the domain the backend server is hosted on. If ownCloud resources should be accessible from other domains, e.g. for a separate web frontend operated on a different domain, administrators can now globally specify policy exceptions via *CORS (Cross-Origin Resource Sharing)* using ``'cors.allowed-domains'`` in *config.php*. Please check *config.sample.php* for more information.
+
+Śolved known issues
+~~~~~~~~~~~~~~~~~~~
+
+- Bogus "Login failed" log entries have been removed (see 10.0.7 known issues)
+- The *Provisioning API* can now properly set default or zero quota
+- User quota settings can be queried through *Provisioning API*
+- A regression preventing a user from setting their e-mail address in the settings page has been fixed
+- File deletion as guest user works again (trash bin permissions are checked correctly)
```

Comment:
it never worked before, wasn't a regression

well, deletion did work but the web UI did not know and needed refresh

Reply:
suggestion: s/again/correctly/

Reply:
done